orttraining/orttraining/eager/ort_util.h

```diff
@@ -10,21 +10,29 @@
 namespace torch_ort {
 namespace eager {
 
-void CreateMLValue(onnxruntime::AllocatorPtr alloc, 
-                   onnxruntime::MLDataType element_type, 
-                   const std::vector<int64_t>& dims, 
+void CreateMLValue(onnxruntime::AllocatorPtr alloc,
+                   onnxruntime::MLDataType element_type,
+                   const std::vector<int64_t>& dims,
                    OrtValue* p_mlvalue);
 
-void CreateMLValue(void* data_ptr, 
-                   onnxruntime::MLDataType element_type, 
-                   const std::vector<int64_t>& dims, 
-                   const OrtMemoryInfo& memory_info, 
+void CreateMLValue(void* data_ptr,
+                   onnxruntime::MLDataType element_type,
+                   const std::vector<int64_t>& dims,
+                   const OrtMemoryInfo& memory_info,
                    OrtValue* p_mlvalue);
 
-void CreateMLValue(void* data_ptr, 
-                   onnxruntime::MLDataType element_type, 
-                   onnxruntime::TensorShape& shape, 
-                   const OrtMemoryInfo& memory_info, 
+void CreateMLValue(void* data_ptr,
+                   onnxruntime::MLDataType element_type,
+                   onnxruntime::TensorShape& shape,
+                   const OrtMemoryInfo& memory_info,
+                   OrtValue* p_mlvalue);
+
+void CreateMLValue(void* data_ptr,
+                   const OrtMemoryInfo& memory_info,
+                   onnxruntime::MLDataType element_type,
+                   const std::vector<int64_t>& dims,
```

Comment:
>const std::vector<int64_t>& d [](http://example.com/codeflow?start=19&length=29)

For input use gsl::span<int64_t const>. This way you can pass any vector, or subspan there